config/checkstyle_checks.xml

```diff
@@ -446,7 +446,8 @@
                          UnsupportedEncodingException, BuildException, ConversionException,
                          FileNotFoundException, TestException, Log, Sets, Multimap,
                          TokenStreamRecognitionException, RecognitionException,
-                         TokenStreamException, IOException"/>
+                         TokenStreamException, IOException, Override, Deprecated, SafeVarargs,
+                         SuppressWarnings"/>
```

Comment:
Please place all new annotation types to new line and you will notice that you missed one default type

Reply:
added